e2e/e2e_basic_test.go

```diff
@@ -1,8 +1,9 @@
-package httpexpect
+package e2e
 
 import (
 	"crypto/tls"
 	"encoding/json"
+	"github.com/gavv/httpexpect/v2"
 	"net/http"
 	"net/http/httptest"
 	"testing"
```

Comment:
Nitpick: let's move httpexpect import to the second import block, like this:

```
import (
	"crypto/tls"
	"encoding/json"
	"net/http"
	"net/http/httptest"
	"testing"

	"github.com/gavv/httpexpect/v2"
	"github.com/valyala/fasthttp/fasthttpadaptor"
)
```

Here and in other files.